docs/camera_example.md

```diff
@@ -63,3 +63,86 @@ python3 camera_client.py --grpc_address [hostname] --grpc_port 9001 --video_sour
 ```
 
 > **NOTE**: Video source is cropped to 704x704 resolution to match model input size.
+
+## Camera frames processing using OCR pipeline
+Optical Character Recognition (OCR) pipeline based on [horizontal text detection](https://docs.openvino.ai/latest/omz_models_model_horizontal_text_detection_0001.html) model, [text recognition](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/text-recognition-0014) 
+combined with a custom node implementation can be used with this client. Using such pipeline each processed frame containes detected boxes and additionaly recognized texts.
+
+> **NOTE**: In the next steps we assume that horizontal text detection model is already downloaded.
+
+### Download text recognition model from OpenVINO Model Zoo
+```bash
+curl -L --create-dir https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/3/text-recognition-0014/FP32/text-recognition-0014.bin -o text-recognition/1/model.bin https://storage.openvinotoolkit.org/repositories/open_model_zoo/2021.4/models_bin/3/text-recognition-0014/FP32/text-recognition-0014.xml -o text-recognition/1/model.xml
+```
+
+```bash
+tree horizontal-text-detection/
+text-recognition
+└── 1
+    ├── model.bin
+    └── model.xml
+```
+
+### Building the Custom Node "horizontal_ocr" Library
+Custom nodes are loaded into OVMS as dynamic library implementing OVMS API from custom_node_interface.h. It can use OpenCV libraries included in OVMS or it could use other thirdparty components.
+
+The custom node horizontal_ocr can be built inside a docker container via the following procedure:
+- go to the custom node source code folder [src/custom_nodes/horizontal_ocr](../src/custom_nodes/horizontal_ocr)
+- run `make` command
+
+This command will export the compiled library in `./lib` folder.
+Copy this `lib` folder to the same location with `horizontal-text-detection` and `text-recognition`.
+
+### OVMS Configuration File
+
+The configuration file for running the camera client with OCR pipeline is stored in [config.json](../src/custom_nodes/horizontal_ocr/config.json)
+Copy this file along with the model files and the custom node library like presented below:
+```bash
+OCR
+├── config.json
+├── text-recognition
+│   └── 1
+│       ├── model.bin
+│       └── model.xml
+├── horizontal-text-detection
+│   └── 1
+│       ├── model.bin
+│       └── model.xml
+└── lib
+    └── libcustom_node_horizontal_ocr.so
+```
+
+### Deploying OVMS
+Deploy OMVS with OCR pipeline using the following command:
+```bash
+chmod -R 755 OCR/
+docker run -p 9001:9001 -d -v ${PWD}/OCR:/OCR openvino/model_server --config_path /OCR/config.json --port 9001
+```
+
+### Requesting the Service
+In the context of [example_client](../example_client) directory, run the following commands.
+
+Install python libraries:
+```bash
+pip3 install -r client_requirements.txt
+```
+
+Start the client
+```bash
+python3 camera_client.py --grpc_address [hostname] --grpc_port 9001 --use_case ocr
+
```

Comment:
maybe some screenshot or short video capture would be handy?

Reply:
Added a gif presenting new camera client usage